### ISSUE TITLE
Details for training Places2 dataset and irregular mask 

### ISSUE THREAD

**Issue by Emy-cv** [geekyutao/RN#32]
Hi, thank you very much for your work.
I have some questions about training details of Places2 dataset:
1. Do you use all the 365 scenes for training? How many epoches did you train for convergence?
2. Do you use the 12000 irregular masks for both training and testing? As I noticed they have a particular training irregular mask dataset.
Thank you very much for your time.

**Comment by sanicle** [comment]
I use all 365 standard data to train with full irregular masks. 
I don't know if my problem is associated with this.
all the 365 scenes？
do you mean resize large pictures to 256*256? is that necessary?
I trained 4,5,6 epochs with single GPU.
GeekTao's pretrained model is 3 epoch multi-gpu model.

**Comment by geekyutao** [comment]
> Hi, thank you very much for your work.
> I have some questions about training details of Places2 dataset:
> 
> 1. Do you use all the 365 scenes for training? How many epoches did you train for convergence?
> 2. Do you use the 12000 irregular masks for both training and testing? As I noticed they have a particular training irregular mask dataset.
>    Thank you very much for your time.

Hi, for 1, yes and 3 epochs(more epochs perhaps better); for 2, yes and check this issue https://github.com/geekyutao/RN/issues/6.

**Comment by Emy-cv** [comment]
> > Hi, thank you very much for your work.
> > I have some questions about training details of Places2 dataset:
> > 
> > 1. Do you use all the 365 scenes for training? How many epoches did you train for convergence?
> > 2. Do you use the 12000 irregular masks for both training and testing? As I noticed they have a particular training irregular mask dataset.
> >    Thank you very much for your time.
> 
> Hi, for 1, yes and 3 epochs(more epochs perhaps better); for 2, yes and check this issue #6.

I see. Thank you very much for your quick respond.

**Comment by Emy-cv** [comment]
> I use all 365 standard data to train with full irregular masks.
> I don't know if my problem is associated with this.
> all the 365 scenes？
> do you mean resize large pictures to 256*256? is that necessary?
> I trained 4,5,6 epochs with single GPU.
> GeekTao's pretrained model is 3 epoch multi-gpu model.

Thank you for your respond, I got the answer from Tao.
